.github/workflows/test.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-13, windows-latest]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
I would suggest, let's not reconfigure the mac VM in this PR, but only separately - now the CI also fails.

Reply:
Yeah we should isolate that I am experimenting this in https://github.com/jdb78/pytorch-forecasting/pull/1648
Still haven't found the working solution.